src/main/java/org/junit/runners/ParentRunner.java

```diff
@@ -104,6 +104,13 @@ protected TestClass createTestClass(Class<?> testClass) {
     protected abstract Description describeChild(T child);
 
     /**
+     * Fires test ignore through {@code notifier} for child corresponding to
+     * {@code child}, which can be assumed to be an element of the list returned
+     * by {@link ParentRunner#getChildren()}.
+     */
+    protected abstract void fireIgnoreChild(T child, RunNotifier notifier);
```

Comment:
We can't add new abstract methods to `ParentRunner`, because there's code outside of JUnit that extends `ParentRunner` directly. Would it be reasonable for this method to do nothing by default?